ddp-workspace/projects/toolkit/src/lib/components/dialogs/sessionWillExpire.component.ts

```diff
@@ -18,29 +17,33 @@ import { finalize, take } from 'rxjs/operators';
                 <span translate>Toolkit.Dialogs.SessionWillExpire.RenewalFailed</span>
             </ng-template>
         </h1>
-        <button mat-icon-button (click)="closeDialog()" [disabled]="isRenewing">
+        <button mat-icon-button (click)="closeDialog()" [disabled]="blockUI">
             <mat-icon class="ddp-close-button">clear</mat-icon>
         </button>
     </div>
     <mat-dialog-content *ngIf="!renewalFailed">
         <p class="Modal-text" translate>Toolkit.Dialogs.SessionWillExpire.Text</p>
     </mat-dialog-content>
+    <mat-dialog-content *ngIf="blockUI">
+        <mat-progress-bar class="Modal-progress" mode="indeterminate"></mat-progress-bar>
+    </mat-dialog-content>
```

Comment:
With a slow Internet connection, the token renewal process can take some time, let's show to a user that something happens.